src/root/Routes.jsx

```diff
@@ -19,6 +20,7 @@ const Routes = props => (
 
             <Route exact path="/signin" component={SignIn} />
             <Route exact path="/signup" component={SignUp} />
+            <Route path='*' exact={true} component={Error404} />
```

Comment:
Need to move below all routes. With this arrangement, the user will not be able to switch to `/dashboard` and` /builder`.
It is also better to stick to one option of quotation marks. In the project, this is `"..."`.